src/javascripts/ng-admin/Crud/misc/EntryFormater.js

```diff
@@ -0,0 +1,57 @@
+/*global define*/
+
+define(function () {
+    'use strict';
+
+    function EntryFormater() {
+    }
+
+    EntryFormater.format = function format(fields) {
+        var map = [];
+        for (var attr in fields) {
+            if (!fields.hasOwnProperty(attr)) {continue;}
+            var order = fields[attr].order();
+            switch (fields[attr].type()) {
+                case 'number':
+                case 'text':
+                case 'wysiwyg':
+                case 'string':
+                case 'boolean':
+                case 'email':
+                case 'json':
+                case 'date':
+                case 'choice':
+                case 'choices':
+                case 'file':
+                case 'template':
+                    map[order] = {name: attr, type: 'Field'};
+                    break;
+                case 'Reference':
```

Comment:
must be `reference` with a lowercase `c`. Same for `referenced_list` (refs #307).